### ISSUE TITLE
Better support for terminal

### ISSUE THREAD

**Issue by jonathanchu** [jonathanchu/atom-one-dark-theme#6]
As reported by @feifanzhou:

<img width="610" alt="screen shot 2015-10-31 at 12 02 17" src="https://cloud.githubusercontent.com/assets/66164/10887490/4c43d294-8155-11e5-8920-49af55ae4ad3.png">

**Comment by feifanzhou** [comment]
I'm running OS X El Cap, Emacs 24.5 (installed via Homebrew) running in iTerm 2.1. The issue happens even when I set my iTerm theme to Atom One.

**Comment by howdoicomputer** [comment]
Same.

**Comment by ghost** [comment]
agree with above

**Comment by jvoisin** [comment]
Same for Xubuntu.
It would be great to force the background colour.

**Comment by st3fan** [comment]
Same for me. Is there a fix for this?

**Comment by iporsut** [comment]
From this http://stackoverflow.com/questions/63950/how-to-make-emacs-terminal-colors-the-same-as-emacs-gui-colors

I think because Terminal supports colors.
We must create color pallate specific for terminal.

**Comment by praveenperera** [comment]
Any updates on this, same problem with iTerm

**Comment by praveenperera** [comment]
Found a solution you can add this code to change the background colours to one of the 256 colours only when running in non-windowed mode.

```lisp
 ;; Customize background for terminal for atom one dark color
  (if (not window-system)
    (atom-one-dark-with-color-variables
      (custom-theme-set-faces
      'atom-one-dark
      `(default ((t (:background "gray14"))))
      `(linum ((t (:foreground "#262626" :background "#262626"))))
      `(hl-line ((t (:background "gray13"))))
      ))
    )
```

**Comment by mdashx** [comment]
Almost there for me... that blue highlighting in the helm buffer doesn't seem right.

![screenshot 2017-01-10 at 9 40 46 pm](https://cloud.githubusercontent.com/assets/7584611/21834765/7ff16442-d77d-11e6-9afe-71c5350a5ab9.png)

**Comment by praveenperera** [comment]
@mdashx try this in the user top of the user-init, instead of what I had above.

```lisp
  (if (not window-system)
      (defvar atom-one-dark-colors-alist
        '(("atom-one-dark-accent"   . "#528BFF")
          ("atom-one-dark-fg"       . "#ABB2BF")
          ("atom-one-dark-bg"       . "gray14")
          ("atom-one-dark-bg-1"     . "gray13")
          ("atom-one-dark-bg-hl"    . "gray13")
          ("atom-one-dark-gutter"   . "#666D7A")
          ("atom-one-dark-accent"   . "#AEB9F5")
          ("atom-one-dark-mono-1"   . "#ABB2BF")
          ("atom-one-dark-mono-2"   . "#828997")
          ("atom-one-dark-mono-3"   . "#5C6370")
          ("atom-one-dark-cyan"     . "#56B6C2")
          ("atom-one-dark-blue"     . "#61AFEF")
          ("atom-one-dark-purple"   . "#C678DD")
          ("atom-one-dark-green"    . "#98C379")
          ("atom-one-dark-red-1"    . "#E06C75")
          ("atom-one-dark-red-2"    . "#BE5046")
          ("atom-one-dark-orange-1" . "#D19A66")
          ("atom-one-dark-orange-2" . "#E5C07B")
          ("atom-one-dark-gray"     . "#3E4451")
          ("atom-one-dark-silver"   . "#AAAAAA")
          ("atom-one-dark-black"    . "#0F1011"))
        "List of Atom One Dark colors.")
    )
```

**Comment by mdashx** [comment]
heck yea @praveenperera - it looks beautiful now! Thank you!

**Comment by jakubdonovan** [comment]
@praveenperera Whenever i paste it into the theme config file, it gives me an error while i load the theme. The error i get "end of file during parsing" any ideas?

**Comment by jhacksworth** [comment]
Can this be closed, now that f677222 is merged?

**Comment by jonathanchu** [comment]
@jhacksworth yes! And thank you again for all your great work into this 👍
